CONTRIBUTING.md

```diff
@@ -34,7 +56,7 @@ You should also add yourself to the `CONTRIBUTORS.md` file in the root of the pr
 ### Reporting Issues
 
 * Make sure you have a [GitHub account](https://github.com/signup/free)
-* Submit a [Github issue](https://github.com/samvera/questioning_authority/issues) by:
+* Submit a [Github issue](https://github.com/samvera/{{library}}/issues/) by:
```

Comment:
@jeremyf I'm wondering if this change works as expected.  I tried it out at Hyrax' [CONTRIBUTING.md](https://github.com/samvera/%7B%7Blibrary%7D%7D/issues/) and it leads to a 404.

Reply:
This does not work as is.